srcpkgs/sov/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'sov'
+pkgname=sov
+version=0.9b
+revision=1
+archs="x86_64"
+build_style=meson
+hostmakedepends="pkg-config wayland-devel"
+makedepends="wayland-devel wayland-protocols freetype-devel libpng-devel glew-devel libxkbcommon-devel liberation-fonts-ttf"
+short_desc="Overlay that shows schemas for all workspaces in sway wm"
+maintainer="Milan Toth <milgra@milgra.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/milgra/sov"
+changelog="https://github.com/milgra/sov/release"
```

Comment:
We prefer a plaintext log; this can be omitted if there isn't one.

Reply:
There is no plaintext log, only the github release descriptions :(

Reply:
If there is no plaintext log, releases is better than nothing, but please fix the typo

Reply:
typo. `releases`, not release